agent/wsclient/client.go

```diff
@@ -48,6 +49,9 @@ const (
 	// wsConnectTimeout specifies the default connection timeout to the backend.
 	wsConnectTimeout = 30 * time.Second
 
+	// wsHandshakeTimeout specifies the default handshake timeout for the websocket client
+	wsHandshakeTimeout = wsConnectTimeout
```

Comment:
If we have wsHandshakeTimeout, do we still need wsConnectTimeout? As if the hand shake timed out, the connect will fail, is that right? Or they should be the same value, eg: `wsHandshakeTimeout 
 = 3*wsConnectTimeout`

Reply:
I would assume the `wsHandshakeTimeout` would be <= `wsConectTimeout`